### PR TITLE
missing_fat_arrows: added strict-mode option, defaults to false

### PR DESCRIPTION
**Pull request by Zolmeister**
see https://github.com/clutchski/coffeelint/pull/391

### CODE REVIEW

**Comment by Zolmeister** [comment]
Just to give a bit more context, this came out of the need for bound callback methods used in Zorium: https://zorium.org/

**Comment by gerry3** [comment]
@Zolmeister is the commit message and pull supposed to say `missing_fat_arrows` instead of `max_line_length`?

File: src/rules/missing_fat_arrows.coffee [hunk 1]
```diff
@@ -15,12 +15,14 @@ module.exports = class MissingFatArrows
     rule:
         name: 'missing_fat_arrows'
         level: 'ignore'
+        isStrict: false
```
Comment (gerry3) [inline src/rules/missing_fat_arrows.coffee]: configuration is supposed to use snake_case

**Comment by Zolmeister** [comment]
Updated, and re-named commit

**Pull request merged**